src/util/navigation.js

```diff
@@ -255,6 +255,29 @@ export function getNetworkSlugInPath (pathname) {
   return get('params.networkSlug', match)
 }
 
+export function getViewContextInPath (pathname) {
```

Comment:
i think this is ok for now, but it feels like a better solution might be to add the :view route parameter to all the routes in PrimaryLayout (we already do this for all the map routes), then we dont need to keep this helper function updated as we change routes

Reply:
Was saying the same above, though more verbosely :) I don't want to hold things up, but I do think it'd be best--and for all the reasons you're citing Pospi--to make this adjustment now. 

Also agree that the other inline links back to community should be considered. I just did the diligence on mobile for this and it'd be nice if a reliable things happened across both apps. We could create a new ticket for that if it feels like this one is scope creeping.